### PR TITLE
Chore: Change to Accordion Component in Edit Settings Page

### PR DESCRIPTION
**Pull request by RakeshUP**
## Description

- Edit Settings page is changed according to the new UI
- An AlertInline was added to the Settings page

Task: [APP-821](https://aragonassociation.atlassian.net/browse/APP-821)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

### CODE REVIEW

File: packages/web-app/src/pages/editSettings.tsx [hunk 1]
```diff
@@ -357,6 +340,10 @@ const EditSettings: React.FC = () => {
             label={t('settings.resetChanges')}
```
Comment (Fabricevladimir) [inline packages/web-app/src/pages/editSettings.tsx:356]: Both buttons should be disabled and enabled using the same condition. Also, can you update to "Reset all changes" per designs?

File: packages/web-app/src/pages/editSettings.tsx [hunk 2]
```diff
@@ -277,68 +291,36 @@ const EditSettings: React.FC = () => {
...
+            type="action-builder"
+            name="metadata"
+            methodName={t('labels.review.daoMetadata')}
+            alertLabel={
+              isMetadataChanged ? t('settings.newSettings') : undefined
+            }
+            dropdownItems={metadataAction}
+          >
+            <AccordionContent>
+              <DefineMetadata />
```
Comment (Fabricevladimir) [inline packages/web-app/src/pages/editSettings.tsx]: The links section of `DefineMetadata` got a styling update. ui-50 for background and secondary button with bgwhite set to false

File: packages/web-app/src/pages/settings.tsx [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import {
```
Comment (Fabricevladimir) [inline packages/web-app/src/pages/settings.tsx:1]: I know this is probably not in the scope of the ticket, but could you quickly add that page header if it doesn't take much time? If not, we can have it as a design debt ticket.

**Review by Fabricevladimir — COMMENTED** [review]
Thanks, Rakesh! Left a couple of comments

**Review by Fabricevladimir — APPROVED** [review]
Good to merge.

File: packages/web-app/src/pages/editSettings.tsx [hunk 4]
```diff
@@ -247,6 +247,32 @@ const EditSettings: React.FC = () => {
     return <Loading />;
   }
 
+  const metadataAction = [
+    {
+      component: (
+        <ListItemAction
+          title={t('labels.resetAction')}
```
Comment (WissenIstNacht) [inline packages/web-app/src/pages/editSettings.tsx]: Wrong label :p

Should be "Reset Changes".

**Review by WissenIstNacht — APPROVED** [review]
Thanks, looks really nice!

We even go lucky with the conflicts. The only only ones are related to the `arrayName` prop I introduced on my PR. Should be easy enough to keep both of them, though.

**Pull request merged**